deepedit/ignite/README.md

```diff
@@ -70,6 +70,24 @@ python ./train.py
     --output      deepedit_model/ \
     --epochs      100
 
+# Train a DeepEdit model using multi gpu
+
+# Using new PyTorch API
+torchrun --standalone \
+       --nnodes=1 \
+       --nproc_per_node=`nvidia-smi -L | wc -l` \
+       --node_rank=0 \
+       --master_addr="localhost" \
+       --master_port=1234 train.py --input /PATH_TO_DATASET/ --output deepedit_model/ --epochs 100 --multi_gpu True
```

Comment:
I think no need the `master_addr`, `master_port`, `node_rank` args for the `standalone` running?
Refer to: https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py#L80

Thanks.

Reply:
agree